app/src/main/java/com/example/una/CharityNavigatorClient.java

```diff
@@ -0,0 +1,54 @@
+package com.example.una;
+
+import android.content.Context;
+
+import com.loopj.android.http.AsyncHttpClient;
+import com.loopj.android.http.JsonHttpResponseHandler;
+import com.loopj.android.http.RequestParams;
+
+
+public class CharityNavigatorClient {
```

Comment:
I like that you not only moved this to a separate class but refactored the other usages to get categories and featured to use this code